{'alpha': 0.0001, 'est': 'br', 'hist_agg': 'none', 'l1_ratio': 0.4367639989, 'n_cases': 17494, 'n_features': 477, 'n_games': 7}
SVD did not converge
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 19, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 779, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 759, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'alpha': 1e-05, 'est': 'br', 'hist_agg': 'none', 'l1_ratio': 0.2521802763, 'n_cases': 30114, 'n_features': 477, 'n_games': 7}
SVD did not converge
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 19, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 779, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 759, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


